src/TurbulenceClosures/turbulence_closure_implementations/anisotropic_minimum_dissipation.jl

```diff
@@ -7,38 +7,32 @@ Parameters for the "anisotropic minimum dissipation" turbulence closure for larg
 proposed originally by [Rozema15](@cite) and [Abkar16](@cite), and then modified
 by [Verstappen18](@cite), and finally described and validated for by [Vreugdenhil18](@cite).
 """
-struct AnisotropicMinimumDissipation{TD, FT, PK, PN, K, PB} <: AbstractEddyViscosityClosure{TD, ThreeDimensionalFormulation}
+struct AnisotropicMinimumDissipation{TD, PK, PN, PB} <: AbstractScalarDiffusivity{TD, ThreeDimensionalFormulation}
     Cν :: PN
     Cκ :: PK
     Cb :: PB
-     ν :: FT
-     κ :: K
 
-    function AnisotropicMinimumDissipation{TD, FT}(Cν::PN, Cκ::PK, Cb::PB, ν, κ) where {TD, FT, PN, PK, PB}
-        κ = convert_diffusivity(FT, κ, Val(false))
-        K = typeof(κ)
-        return new{TD, FT, PK, PN, K, PB}(Cν, Cκ, Cb, ν, κ)
+    function AnisotropicMinimumDissipation{TD}(Cν::PN, Cκ::PK, Cb::PB) where {TD, PN, PK, PB}
+        return new{TD, PK, PN, PB}(Cν, Cκ, Cb)
     end
 end
 
 const AMD = AnisotropicMinimumDissipation
 
-Base.show(io::IO, closure::AMD{TD, FT}) where {TD, FT} =
-    print(io, "AnisotropicMinimumDissipation{$FT} turbulence closure with:\n",
+@inline viscosity(::AMD, diffusivities, args...) = diffusivities.νₑ
```

Comment:
is this not a `[i, j, k]` field?

Reply:
It is, but this function just returns the _object_; it's indexed into later.

I think we can further simplify the interface...

Reply:
See here:

https://github.com/CliMA/Oceananigans.jl/blob/cb97515799b6889b955fdc36474e47f97c0a68d1/src/TurbulenceClosures/vertically_implicit_diffusion_solver.jl#L139-L176

We also would use this function for immersed boundary fluxes.

I might try to clean this up now though.